### ISSUE TITLE
[Bug]: Broken project description after 'mlrun.load_project'

### ISSUE THREAD

**Issue by george0st** [mlrun/mlrun#5052]
### MLRun Version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the latest version of MLRun CE.

### Reproducible Example

```python
import mlrun
import mlrun.feature_store as fstore
from mlrun.datastore.targets import RedisNoSqlTarget,SQLTarget
from mlrun.data_types.data_types import ValueType

def invalid_description():

    mlrun.set_env_from_file("mlrun-nonprod.env")

    project_name="a1"
    project = mlrun.get_or_create_project(project_name, context='./', user_project=False)
    project.description = "a1 description"
    project.save()

    project_name="a2"
    project = mlrun.get_or_create_project(project_name, context='./', user_project=False)
    project.description = "a2 description"
    project.save()

    project_name="a3"
    project = mlrun.get_or_create_project(project_name, context='./', user_project=False)
    project.description = "a3 description"
    project.save()

    mlrun.load_project(name="a1", context="./", user_project=False)
    
    # error, project "a1" has description "a3 description", after load_project

if __name__ == '__main__':
    invalid_description()
```

### Issue Description

See sample, the error happened, after load_project, because project "a1" has finally description "a3 description" (and it is wrong)
The issue is, that probably **mlrun.load_project** changed project description see picture from UI (the same output return http://[localhost:8080/api/v1/projects](http://localhost:8080/api/v1/projects)

![image](https://github.com/mlrun/mlrun/assets/95856749/c1fe5055-ed2e-4568-aac5-08b67604a362)

See also
```
 "projects": [
        {
            "kind": "project",
            "metadata": {
                "name": "a1",
                "created": "2024-02-03T13:59:29.656597",
                "labels": {},
                "annotations": {}
            },
            "spec": {
                "description": "a3 description",
                "owner": null,
                "goals": null,
                "params": {},
                "functions": [],
                "workflows": [],
                "artifacts": [],
                "artifact_path": null,
                "conda": "",
                "source": "",
                "subpath": null,
                "origin_url": null,
                "desired_state": "online",
                "custom_packagers": [],
                "default_image": null,
                "build": {
                    "commands": [],
                    "requirements": []
                }
            },
            "status": {
                "state": "online"
            }
        },
```

### Expected Behavior

The  **mlrun.load_project** cannot change the project parameters such as description, etc.

### Installation OS

Windows

### Installation Method

Docker

### Python Version

3.9.10

### MLRun Version

1.5.2

**Comment by liranbg** [comment]
Hi @george0st - I am trying to understand, under what circumstances would you save & load many projects under same directory assuming they wont interfere eachother
meaning, when you provide `"context='./'"` you essentially tell what "base" project.yaml to use when creating/importing.

perhaps, if you would specify different `context` directories this will never happen.
tl;dr: project context directory is meant for a single project, not many - or else you will experience exactly that

**Comment by george0st** [comment]
Hi @liranbg , when I used different content path for each project, it works, thx. 

My first idea was wrong, because I expected than project path contains content path plus project name, but reality was different (I kept in head path in v3io and it seems little different than path in standard file system).

Your advice help me, I will close this issue.
